src/plugins/vis_default_editor/public/_default.scss

```diff
@@ -91,8 +91,7 @@
   }
 
   .visEditor__visualization__wrapper {
-    // force the visualization to take 100% width and height.
+    // force the visualization to take 100% width.
     width: 100% !important;
-    height: 100% !important;
```

Comment:
@stratoula  I found one regression on chrome on removing the height property 

![image](https://user-images.githubusercontent.com/20072247/113701050-4b979c80-96e0-11eb-81d3-7311fa06f1cb.png)

Reply:
Thanx Alex, it should be ok now